include/config/pokemon.h

```diff
@@ -51,6 +51,8 @@
 #define P_TWO_FRAME_FRONT_SPRITES        TRUE        // In Pokémon Emerald, Pokémon front sprites always consist of two frames. This config can revert it to only use the first frame, as is the case in the other Gen 3 games.
 #define P_ONLY_OBTAINABLE_SHINIES        FALSE       // If TRUE, Pokémon encountered in the Battle Pyramid won't be shiny.
 #define P_NO_SHINIES_WITHOUT_POKEBALLS   FALSE       // If TRUE, Pokémon encountered when the player is out of Poké Balls won't be shiny
+#define P_DYNAMIC_HIDDEN_POWER_TYPES     FALSE       // If TRUE, Pokémon will display their Hidden Power type in summary screens and in battle.
+#define P_DYNAMIC_IVY_CUDGEL             FALSE       // If TRUE, Ivy Cudgel type will display in summary screens according to Ogerpon's held mask.
```

Comment:
I would combine those. There is no reason to have one but not the other in my opinion. We can also use it for moves like Raging Bull